iroh-net/src/magicsock/endpoint.rs

```diff
@@ -149,9 +149,23 @@ impl Endpoint {
         }
     }
 
+    /// Returns the derp addr of this endpoint
+    pub fn derp_addr(&self) -> Option<u16> {
+        self.derp_addr
+    }
+
+    /// Adds a derp addr for this endpoint
+    pub fn add_derp_addr(&mut self, region: u16) {
+        self.derp_addr = Some(region);
+    }
+
     /// Returns the address(es) that should be used for sending the next packet.
     /// Zero, one, or both of UDP address and DERP addr may be non-zero.
     fn addr_for_send(&mut self, now: &Instant) -> (Option<SocketAddr>, Option<u16>, bool) {
+        if std::option_env!("DEV_DERP_ONLY").is_some() {
+            trace!("in `DEV_DERP_ONLY` mode, giving the DERP address as the only viable address for this endpoint");
```

Comment:
I'm moving this particular one to `debug` because it happens so often that if we printed it as `info` it would flood the user.

The others are moved to `warn`. How does that sound?